### PR TITLE
Remove needless OpcredsIssuer dependency

### PR DESCRIPTION
**Pull request by tcarmelveilleux**
#### Problem
- Dependency on ExampleOperationalCredentialsIssuer was in
  many files where not needed, making other refactors more
  difficult.

#### Change overview
Remove this dependency and make CHIPCommand own its ephemeral
operational credentials issuer.

#### Testing

- Unit tests and integration tests still pass

### CODE REVIEW

**Review by msandstedt — APPROVED** [review]

**Review by emargolis — APPROVED** [review]

**Review by jmeg-sfy — APPROVED** [review]

File: examples/chip-tool/commands/common/CHIPCommand.cpp [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 #include "CHIPCommand.h"
 
+#include "controller/ExampleOperationalCredentialsIssuer.h"
```
Comment (andy31415) [inline examples/chip-tool/commands/common/CHIPCommand.cpp:21]: other includes here use `<>` instead of `""`. Could we do the same?

**Review by andy31415 — APPROVED** [review]

File: examples/chip-tool/commands/common/CHIPCommand.cpp [hunk 2]
```diff
@@ -39,6 +40,8 @@ constexpr chip::FabricId kCommissionerGammaFabricId = 3;
 
 CHIP_ERROR CHIPCommand::Run()
 {
+    chip::Controller::ExampleOperationalCredentialsIssuer opCredsIssuer;
```
Comment (bzbarsky-apple) [inline examples/chip-tool/commands/common/CHIPCommand.cpp:43]: This is not the scope in which this thing is used...  It really does need to be member variable of CHIPCommand for the lifetime to be right, no?

File: examples/chip-tool/commands/common/CHIPCommand.h [hunk 3]
```diff
@@ -82,7 +82,6 @@ class CHIPCommand : public Command
     static void RunQueuedCommand(intptr_t commandArg);
 
     CHIP_ERROR mCommandExitStatus = CHIP_ERROR_INTERNAL;
-    chip::Controller::ExampleOperationalCredentialsIssuer mOpCredsIssuer;
```
Comment (bzbarsky-apple) [inline examples/chip-tool/commands/common/CHIPCommand.h:85]: As far as I can tell, this needs to stay....

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]

File: examples/chip-tool/commands/common/CHIPCommand.h (hunk 3 above)
Comment (tcarmelveilleux) [reply examples/chip-tool/commands/common/CHIPCommand.h:85]: This needs to stay based on refactors from @vivien-apple that were merged after I was done. It caused a merge issue I did not catch right away. It's fixed now.

**Review by bzbarsky-apple — APPROVED** [review]
Not sure why the member needs to move from private to protected.

**Comment by tcarmelveilleux** [comment]
> Not sure why the member needs to move from private to protected.

To match other similarly scoped items just as storage

**Pull request merged**